internal/persistence/sql/migrations/templates/20210623162417_relationtuple.mysql.up.sql

```diff
@@ -0,0 +1,36 @@
+-- we can not fizz this migration because there is no check constraint support in fizz
+-- and it gets cumbersome with sqlite; having a working SQL version is actually way easier
+CREATE TABLE keto_relation_tuples
+(
+    shard_id                 char(36)    NOT NULL,
+    nid                      char(36)    NOT NULL,
+    namespace_id             char(36)    NOT NULL,
```

Comment:
This is missing a FK relation to namespace

Reply:
good catch